modules/functions.js

```diff
@@ -1,6 +1,7 @@
-/* eslint-disable no-use-before-define */
 /* eslint-disable no-plusplus */
```

Comment:
Great job @huzaifakhan0308 🤝

- [ ] [Optional] As the previous reviewer stated, kindly do not disable linters as they help us write quality and error-free code. Kindly do this across all your Javascript files.
- [ ] [Optional] Your project is not implementing the intended functionality yet. I am not able to add and remove books from the list of books.
- [ ] [Optional] Kindly make sure it is possible to navigate through your nav links of `List`, `Add New`, and `Contact`.